examples/cars/configs/train_cars.yaml

```diff
@@ -37,6 +37,7 @@ metric_args:
   metrics_to_exclude_from_visualization: [cmc,]
   cmc_top_k: [1]
   map_top_k: [5]
+  fmr_vals: [0.5, 0.1, 0.01, 0.001]
```

Comment:
Could you please find the most frequently used values of these metrics in the literature? 
For example, for the metrics we have, these values are `[1, 5, 10]`

Reply:
In face recognition papers the most common values are 0.01% and 0.0001%. For fingerprints recognition I met 0% and 1%. I think that FMR values are really depends on the domain: in security-related domains it should be as small as possible, in other domains it could be much bigger. I think that 1% or 5% as a default value will be a good choice.

Reply:
Got it. Then let's replace the values in configs with those that you think are the good default choice

Reply:
I set just 1%, because for the other metrics mostly only one `k` was set.

Reply:
Yes, that's also fine